extensions-core/hdfs-storage/src/test/java/org/apache/druid/indexing/common/tasklogs/HdfsTaskLogsTest.java

```diff
@@ -95,6 +95,19 @@ public void test_taskStatus() throws Exception
     );
   }
 
+  @Test
+  public void test_taskPayload() throws Exception
+  {
+    final File tmpDir = tempFolder.newFolder();
+    final File logDir = new File(tmpDir, "logs");
+    final File payload = new File(tmpDir, "payload.json");
+    final TaskLogs taskLogs = new HdfsTaskLogs(new HdfsTaskLogsConfig(logDir.toString()), new Configuration());
+
+    Files.write("{}", payload, StandardCharsets.UTF_8);
```

Comment:
You may update this (and other usages in this file) to use `Files.asCharSink(file, charset).write({})` as suggested by javadoc.